.travis.yml

```diff
@@ -15,7 +15,7 @@ script:
   - yarn screenshots --mode react --viewport desktop
   - yarn screenshots --mode react --viewport 300x600 --no-empty-screenshot-dir
   - yarn screenshots --mode stack --no-empty-screenshot-dir
-  - yarn screenshots --mode kss --no-empty-screenshot-dir
+  - travis_wait yarn screenshots --mode kss --no-empty-screenshot-dir
```

Comment:
Should we add it to other screenshots command? We already had this kind of error before, no?

Reply:
De mémoire c'était sur la même commande. Honnêtement je ne sais pas, en tout cas je garde l'astuce en tête si jamais ça se reproduit

Reply:
Je crois qu'on a déjà parlé de ca, est ce que t'es sur qu'au bout d'un moment, après 10m d'attente, puppeteer se réveille pour prendre d'autres screenshots ?